zetta_utils/mazepa_layer_processing/common/subchunkable_apply_flow.py

```diff
@@ -637,7 +637,7 @@ def _path_join_if_not_none(base: str | None, suffix: str) -> str | None:
 
 def _expand_bbox_resolution(  # pylint: disable=line-too-long
     bbox: BBox3D,
-    dst_resolution: Vec3D,
+    dst_resolution: Vec3D[float],
```

Comment:
Was this causing typing issues? Or just adding more strict typing?

Reply:
No issues with mypy, I think vscode has some issues that I was trying to fix. I don't think this change is necessary. Do you want me to remove it?

Reply:
Yeah - I think looser is better here.